src/main/resources/validationrules/01_validate_request.drl

```diff
@@ -9,14 +8,14 @@ import uk.gov.hmcts.reform.roleassignment.domain.model.enums.RoleType
 import uk.gov.hmcts.reform.roleassignment.domain.model.Pattern;
 import com.fasterxml.jackson.databind.JsonNode;
 import uk.gov.hmcts.reform.roleassignment.domain.model.Role
-import uk.gov.hmcts.reform.roleassignment.domain.model.Case;
 
 
-rule "R11_role_validation_for_organization_pattern" salience 65
+
+rule "R01_role_validation_for_organization_pattern" salience 100
 
 when
 $requestedRole :  RoleAssignment()
-                  RoleAssignment($requestedRole.getStatus() == Status.ROLE_VALIDATED)
+
```

Comment:
In the classification logic below, will it loop for each item provided in classification list in pattern?

Reply:
I am updating the constraints in drool rules as it's part of performance improvement.

Reply:
we need to reorder these conditions based upon pattern, like RoleType then GrantType then classification

Reply:
As, I mentioned earlier, I am updating constraint in rules based on complexity to improve performance.